src/mpid/ch4/src/ch4_globals.c

```diff
@@ -22,6 +22,16 @@ MPIDI_av_table_t *MPIDI_av_table0;
 MPIDI_NM_funcs_t *MPIDI_NM_func;
 MPIDI_NM_native_funcs_t *MPIDI_NM_native_func;
 
+#if defined(MPIDI_CH4_USE_WORK_QUEUES)
+struct MPIDI_workq_elemt MPIDI_workq_elemt_direct[MPIDI_WORKQ_ELEMT_PREALLOC] = { {0}
+};
```

Comment:
Maybe not put the `};` on a new line for cleanness.

Reply:
I totally agree with you but I think this was what the style checker forced me to do. I'll try to see if I can write this in some different way so both humans and the style checker feel comfortable.

Reply:
I see. Maybe we should fix the style checker. Another day.